btr-web/btr-main-app/components/person-info/Citizenship.vue

```diff
@@ -1,3 +1,14 @@
+<template>
+  <div class="flex flex-wrap">
```

Comment:
'class' is the only change here, to follow the design (verified with julie) outside the template flip